packages/apollo-server-core/src/graphqlOptions.ts

```diff
@@ -41,6 +42,7 @@ export interface GraphQLServerOptions<
   // cacheControl?: boolean | CacheControlExtensionOptions;
   cacheControl?: boolean | any;
   extensions?: Array<() => GraphQLExtension>;
+  cache?: KeyValueCache;
```

Comment:
As mentioned on slack I think this should explicitly be `persistedQueries: { cache }` since you may want a different implementation for other caching solutions. Plus it is surprising that passing something to top level "cache" doesn't actually give you a GraphQL cache.